packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.ts

```diff
@@ -148,12 +147,13 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       } seconds), open price: ${format(after?.openPrice)}.
         To see these prices, make a GET request to:
         https://api.cryptowat.ch/markets/${this.exchange}/${this.pair}/ohlc?after=${
-        before?.openTime + this.ohlcPeriod
+        (before?.openTime || 0) + this.ohlcPeriod
```

Comment:
I'm trying to see if this impacts anything. a bad case would be if the feed fails and then defaults to 0 as a result of the `|| 0` but this is only in the `Error` mode. other than this change I really don't think there are any issues introduced in this critical feed.

Reply:
I think this is implicitly what it was doing before since it was previously passing undefined or null to an arithmetic operation. I can change the logic to something else if you think there's something else we should do if this is null/undefined?

Thoughts?

Reply:
no, I think it's all good and does not introduce anything of issue. After looking at it again I think we are good to go. Was just trying to think of potential issues this could introduce.

Reply:
no, I think it's all good and does not introduce anything of issue. After looking at it again I think we are good to go. Was just trying to think of potential issues this could introduce.